src/d/actor/d_a_kita.cpp

```diff
@@ -4,52 +4,448 @@
-    /* Nonmatching */
+    return TRUE;
 }
 
 /* 00001CC0-00001D3C       .text daKita_Delete__FP10kita_class */
-static BOOL daKita_Delete(kita_class*) {
-    /* Nonmatching */
+static BOOL daKita_Delete(kita_class* i_this) {
+    dComIfG_resDelete(&i_this->mPhs, "Kita");
+    if(i_this->heap != NULL){
+        dComIfG_Bgsp()->Release(i_this->pm_bgw);
+    }
+    if(i_this->mBaseEmitter != NULL){
+        i_this->mBaseEmitter->becomeInvalidEmitter();
+    }
+
+    return TRUE;
 }
 
 /* 00001D3C-00001EB0       .text CallbackCreateHeap__FP10fopAc_ac_c */
-static BOOL CallbackCreateHeap(fopAc_ac_c*) {
-    /* Nonmatching */
+static BOOL CallbackCreateHeap(fopAc_ac_c* p_this) {
+    BOOL ret;
+    kita_class* actor = static_cast<kita_class*>(p_this);
+
+    J3DModelData* modelData = static_cast<J3DModelData*>(dComIfG_getObjectRes("Kita", 4));
+    actor->mModel = mDoExt_J3DModel__create(modelData, 0, 0x11020203);
+
+    if(actor->mModel == NULL){
+        ret = FALSE;
+    }
+    else {
+        JUT_ASSERT(0x3b2, modelData != 0);
+        actor->pm_bgw = new dBgW();
+        JUT_ASSERT(0x3b7, actor->pm_bgw != NULL);
+        actor->pm_bgw->Set(static_cast<cBgD_t*>(dComIfG_getObjectRes("Kita", 7)), cBgW::MOVE_BG_e, &actor->mBgwMtx);
+        actor->pm_bgw->SetCrrFunc(dBgS_MoveBGProc_Typical);
+        actor->pm_bgw->SetRideCallback(ride_call_back);
+        ret = TRUE;
+    }
+
+    return ret;
 }
 
 /* 00001EB0-00002224       .text daKita_Create__FP10fopAc_ac_c */
-static cPhs_State daKita_Create(fopAc_ac_c*) {
-    /* Nonmatching */
+static cPhs_State daKita_Create(kita_class* i_this) {
+    /* Nonmatching - regalloc and data address*/
+    static dCcD_SrcSph utiwa_sph_src = {
+        // dCcD_SrcGObjInf
+        {
+            /* Flags             */ 0,
+            /* SrcObjAt  Type    */ 0,
+            /* SrcObjAt  Atp     */ 0,
+            /* SrcObjAt  SPrm    */ dCcG_AtSPrm_StopNoConHit_e | dCcG_AtSPrm_NoConHit_e,
+            /* SrcObjTg  Type    */ AT_TYPE_WIND,
+            /* SrcObjTg  SPrm    */ cCcD_TgSPrm_Set_e | cCcD_TgSPrm_IsEnemy_e,
+            /* SrcObjCo  SPrm    */ 0,
+            /* SrcGObjAt Se      */ 0,
+            /* SrcGObjAt HitMark */ 0,
+            /* SrcGObjAt Spl     */ 0,
+            /* SrcGObjAt Mtrl    */ 0,
+            /* SrcGObjAt SPrm    */ 0,
+            /* SrcGObjTg Se      */ 0,
+            /* SrcGObjTg HitMark */ 0,
+            /* SrcGObjTg Spl     */ 0,
+            /* SrcGObjTg Mtrl    */ 0,
+            /* SrcGObjTg SPrm    */ dCcG_TgSPrm_NoConHit_e,
+            /* SrcGObjCo SPrm    */ 0,
+        },
+        // cM3dGSphS
+        {
+            /* Center */ 0.0f, 0.0f, 0.0f,
+            /* Radius */ 400.0f,
+        },
+    };
+
+    fopAcM_SetupActor(i_this, kita_class);
+    s32 ret = dComIfG_resLoad(&i_this->mPhs, "Kita");
+    if(ret != cPhs_COMPLEATE_e){
+        return ret;
+    }
+
+    i_this->field_2A0 = i_this->base.mParameters;
+    i_this->field_2A1 = i_this->base.mParameters >> 8;
```

Comment:
```suggestion
    i_this->field_2A0 = fopAcM_GetParam(i_this) & 0xFF;
    i_this->field_2A1 = (fopAcM_GetParam(i_this) >> 8) & 0xFF;
```
The devs tend to prefer using inlines for `base` member accesses